src/ray/object_manager/plasma/dlmalloc.cc

```diff
@@ -255,6 +273,7 @@ int fake_munmap(void *addr, int64_t size) {
   return r;
 }
 
+namespace detail {
```

Comment:
Better to use a more specific name, such as `namespace config`

Reply:
ah i was following the folly convention that `detail` space means `private`, don't use it.

Reply:
Why not just calling it private then?

Reply:
private is a reserved keywords lol. so basically facebook use `detail` convention, google use `internal` convention.
now choose your poison.

![image](https://user-images.githubusercontent.com/837180/127545105-9787c5f1-af45-4e86-9d21-67baed1c0d67.png)

Reply:
I will follow the convention of where you are from :)  (detail). Btw, I need to probably change the namespace in the pubsub module to be detail as well in this case... (I used pubusb_internal there lol)

Reply:
actually, let's use internal. it seems we have been using it in a couple of places.